src/modules/events/status_display.h

```diff
@@ -45,7 +45,7 @@
 #include <uORB/Publication.hpp>
 #include <uORB/Subscription.hpp>
 #include <uORB/topics/battery_status.h>
-#include <uORB/topics/cpuload.h>
+#include <uORB/topics/vehicle_cpuload.h>
```

Comment:
If cpuload changes we'll need to update flight review. https://github.com/px4/flight_review

Reply:
Yes, this would be bad. I don't frankly remember what was the reason behind this any more; it was done very early when I checked if there was something HW dependent that could be simply replaced with os-provided interfaces. I'll check again if I can just remove this change.

Reply:
This is indeed not needed. Thanks for drawing my attention to it, I removed this change